pkg/cmd/root/help_topic.go

```diff
@@ -10,10 +10,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var HelpTopics = map[string]map[string]string{
-	"mintty": {
-		"short": "Information about using gh with MinTTY",
-		"long": heredoc.Doc(`
+type helpTopic struct {
```

Comment:
It seemed reasonable to move this to a strongly typed structure rather than a nested map of strings.

Reply:
Ex husband used temp numbers to change on my social profiles  my name is Sonya Hand